alephium/contracts/governance.ral

```diff
@@ -7,7 +7,7 @@ Contract Governance(
     mut guardianSets: [ByteVec; 2], // keep previous and current guardian set
     mut guardianSetIndexes: [U256; 2],
     mut previousGuardianSetExpirationTimeMS: U256
-) extends Constants() {
+) extends Constants(), Upgradable() {
```

Comment:
This is still using `Upgradable`, is it an on-purpose design?

Reply:
Yes, because both `Governance` and `TokenBridge` can be upgraded. To reduce code duplication, I introduced the `Upgradable` interface, and both `Governance` and `TokenBridge` need to extend the `Upgradable` contract.

Reply:
Can `Governance` use the upgrade code from `token_bridge_factory` too?

Reply:
It seems possible, we need to add a field(`tokenBridgeFactory`) for the `Governance` contract.

Reply:
I think it's worth it as a  contract field uses less storage than a function.